publication-commons/src/test/java/no/unit/nva/publication/service/impl/MigrationTests.java

```diff
@@ -122,23 +144,72 @@ void shouldMigrateUncompressedDoiRequestToCompressedAndAddNewFields() throws IOE
         assertThat(doi.getIdentifier(), is(equalTo(doiRequest.get(0).getIdentifier())));
     }
 
+    @Test
+    void shouldAddCuratingInstitutionForVerifiedContributors() {
```

Comment:
Can you specifically test a null EntityDescription and empty Contributors

Reply:
yup!